lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -1,15 +1,24 @@
 # bgp_af.yaml
 ---
 _template:
-  config_get: 'show running bgp all'
   config_get_token: '/^router bgp <asnum>$/'
-  config_get_token_append:
-    - '/^vrf <vrf>$/'
-    - '/^address-family <afi> <safi>$/'
   config_set: "router bgp <asnum>"
-  config_set_append:
-    - 'vrf <vrf>'
-    - 'address-family <afi> <safi>'
+  cli_nexus:
+    config_get: 'show running bgp all'
+    config_get_token_append:
+      - '/^vrf <vrf>$/'
+      - '/^address-family <afi> <safi>$/'
+    config_set_append:
+      - 'vrf <vrf>'
+      - 'address-family <afi> <safi>'
+  cli_ios_xr:
+    config_get: 'show running-config router bgp'
+    config_get_token_append:
+      - '/^vrf <vrf>$/'
+      - '/^address-family <afi> <safi>$/'
```

Comment:
You should be able to share the config_get_token_append between both platforms since it's the same.

```yaml
_template:
  config_get_token: '/^router bgp <asnum>$/'
  config_get_token_append:
      - '/^vrf <vrf>$/'
      - '/^address-family <afi> <safi>$/'
  cli_nexus:
  ...
```